js/src/index.js

```diff
@@ -0,0 +1,56 @@
+const CredentialRequestFactory = require("./credentialRequest/credentialRequestFactory");
+const DownloadFailedException = require("./exception/DownloadFailedException");
+const CredentialResponseFactory = require("./credentialResponse/CredentialResponseFactory");
+const NetworkRequestTimeoutException = require("./exception/NetworkRequestTimeoutException");
+const { NETWORK_TIMEOUT, EMPTY_RESPONSE } = require("./constants/constants");
+const Logger = require("./common/Logger");
+const { default: axios } = require("axios");
+
+/**
+ * @param {Object} issuerMeta
+ * @param {string} proof
+ * @param {string} accessToken
+ */
+
+async function requestCredential(issuerMeta, proof, accessToken) {
+  const request = CredentialRequestFactory.createCredentialRequest(
+    issuerMeta.credentialFormat,
+    accessToken,
+    issuerMeta,
+    proof
```

Comment:
In kotlin / swift we accept Proof object ({proof_type: "jwt",jwt:"<jwt-value>"}) as proof (not just the proof value as proof), is it the same here?

Reply:
We are constructing this object inside the request body based on the credential format and its expected request should we actually force this in higher layer to provide in this type?

Question - Does the request of all issuers have this format {proof_type:"12",12:""} or it differs based on the issuer?